policyengine_us/variables/gov/states/oh/tax/income/credits/oh_retirement_income_credits.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class oh_retirement_income_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Retirement Income Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.055"
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.oh.tax.income.credits.retirement_income
+
+        person = tax_unit.members
+        pension_income = tax_unit.sum(person("pension_income", period))
```

Comment:
I believe we only count the retirement income of the head and the spouse (f.e. if there are old dependents in the tax_unit that have pensions, they dont count)

Reply:
I think this time only the pension of head and spouse are added up